group_vars/all.yml.sample

```diff
@@ -235,6 +235,7 @@ dummy:
 # a URL to the .repo file to be installed on the targets.  For deb,
 # ceph_custom_repo should be the URL to the repo base.
 #
+#ceph_custom_key: https://server.domain.com/ceph-custom-repo-key.asc
```

Comment:
you shouldn't edit this file directly, you must edit `roles/ceph-defaults/defaults/main.yml` and run the `generate_group_vars_sample.sh` script

Reply:
OK